src/types/flags.ts

```diff
@@ -11,7 +11,6 @@ export type Flags = {
   technicalAssistance: boolean;
   hide508Workflow: boolean;
   portfolioUpdateReport: boolean;
-  trbAdminNoteUpdates: boolean;
```

Comment:
Will remove LaunchDarkly FF once this is in production (https://jiraent.cms.gov/browse/EASI-3518)